tests/test_condition.py

```diff
@@ -103,7 +104,144 @@ def verify() -> None:
         assert TRANSLATOR.visit(exp) == translated
 
     if exception is not None:
-        with pytest.raises(type(exception), match=str(exception)):
+        with pytest.raises(type(exception), match=re.escape(str(exception))):
+            verify()
+    else:
+        verify()
+
+
+boolean_tests = [
+    pytest.param(
+        bool_cond(
+            BooleanOp.OR,
+            [Condition(Column("a"), Op.EQ, 1), Condition(Column("a"), Op.EQ, 2)],
+        ),
+        BooleanCondition(
+            BooleanOp.OR,
+            [Condition(Column("a"), Op.EQ, 1), Condition(Column("a"), Op.EQ, 2)],
+        ),
+        "(a = 1 OR a = 2)",
+        None,
+        id="basic boolean test",
+    ),
+    pytest.param(
+        bool_cond(
+            BooleanOp.AND,
+            [
+                Condition(Column("a"), Op.EQ, 1),
+                Condition(Column("b"), Op.EQ, 2),
+                Condition(Column("c"), Op.EQ, 3),
+            ],
+        ),
+        BooleanCondition(
+            BooleanOp.AND,
+            [
+                Condition(Column("a"), Op.EQ, 1),
+                Condition(Column("b"), Op.EQ, 2),
+                Condition(Column("c"), Op.EQ, 3),
+            ],
+        ),
```

Comment:
If we go this way I would make BooleanCondition abstract and provide two subclasses or helper functions:
`And` and `Or`. Imagine something like: `and(condition, condition, condition ....)`

Reply:
I like this idea. I'll add it.